rsts/index.rst

```diff
@@ -3,41 +3,42 @@ Flyte
 
 .. image:: images/flyte_lockup_gradient_on_light.png
 
-Flyte is a structured programming and distributed processing platform created at Lyft that enables highly concurrent, scalable and maintainable workflows for machine learning and data processing.
-Welcome to the documentation hub for Flyte.
-
-.. toctree::
-   :maxdepth: 1
-   :caption: Getting Started
-   :name: gettingstartedtoc
-
-   getting_started/first_run
-   getting_started/first_example
-   getting_started/run_on_flyte
-   getting_started/learn
-
-.. toctree::
-   :caption: How-Tos
-   :maxdepth: 1
-   :name: howtotoc
-
-   plugins/index
-   howto/index
-
-.. toctree::
-   :caption: Deep Dive
-   :maxdepth: 1
-   :name: divedeeptoc
-
-   dive_deep/index
-   reference/index
-
-.. toctree::
-   :caption: Contributor Guide
-   :maxdepth: 1
-   :name: roadmaptoc
-
-   community/index
-   community/docs
-   community/roadmap
-   community/compare
+Meet Flyte
+==========
+
+Flyte is an open-source, container-native, structured programming and distributed processing platform. It enables highly concurrent, scalable and maintainable workflows for machine learning and data processing.
+
+Created at Lyft, Flyte provides first class support for Python, and is built directly on Kubernetes for all the benefits containerization provides: portability, scalability, and reliability.
+
+Flyte provides a single unit of execution (task) as a top-level concept. Multiple tasks arranged in a data producer-consumer order create the workflow, which is pure specification created in any language.
```

Comment:
...which is a pure specification...

Reply:
I believe we can include a short code snippet depicting `task` and `workflow` -- just to visualize the theoretical explanation.

Reply:
Thank you @cosmicBboy for including them below.